src/gfn/gym/helpers/preprocessors.py

```diff
@@ -35,7 +35,6 @@ def __init__(
         self,
         height: int,
         ndim: int,
-        get_states_indices: Callable[[States], TT["batch_shape", "input_dim"]],
```

Comment:
This one i'm not sure of - why did you feel compelled to remove it?

Reply:
Since,
1. It was not used!
2. The parent class `Preprocessor` does not have it as a parameter

Reply:
Oh no!  Great catch if so. Can you ensure this is truly not needed across the entire library? I don't mind if we keep this as part of the current PR.

![Screenshot 2024-10-28 125646](https://github.com/user-attachments/assets/6848d82f-3a37-4673-9ce3-da0afc493ff7)

Reply:
I've checked them thoroughly. The change has no problem, and the other parts seem to have no problem.

Reply:
Thanks!